app/views/layouts/blacklight/base.html.erb

```diff
@@ -1,35 +1,43 @@
 <!DOCTYPE html>
 <html lang="en" class="no-js">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-
-    <!-- Internet Explorer use the highest version available -->
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-
-    <title><%= render_page_title %></title>
-    <%= opensearch_description_tag application_name, opensearch_catalog_url(format: 'xml') %>
-    <%= favicon_link_tag %>
-    <%= stylesheet_link_tag "application", media: "all" %>
-    <%= javascript_include_tag "application" %>
-    <%= csrf_meta_tags %>
-    <%= content_for(:head) %>
-  </head>
-  <body class="<%= render_body_class %>">
-  <%= render partial: 'shared/header_navbar' %>
-
-  <div id="main-container" class="<%= container_classes %>">
-    <%= content_for(:container_header) %>
-
-    <%= render partial: 'shared/flash_msg', layout: 'shared/flash_messages' %>
-
-    <div class="row">
-      <%= content_for?(:content) ? yield(:content) : yield %>
-    </div>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+
+  <!-- Mobile viewport optimization h5bp.com/ad -->
+  <meta name="HandheldFriendly" content="True">
```

Comment:
This appears to revert some commits like https://github.com/projectblacklight/blacklight/commit/42a0fca01bb3a9511a1d92c748dfceeaab31cdcc#diff-01a8fe6697b36c0d2663ca12f0fe9502